src/swap/changelly.js

```diff
@@ -264,7 +268,7 @@ export function makeChangellyPlugin(
         },
         promoCode
       )
-      checkReply(sendReply, request)
+      checkReply(sendReply, request, log)
```

Comment:
Line 193 should have a log noting that the fixed rate quote failed. As well, it should log if the estimate quote also fails